tests/_files/BankAccount-cobertura.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE coverage SYSTEM "http://cobertura.sourceforge.net/xml/coverage-04.dtd">
+<coverage lines-valid="10" lines-covered="5" line-rate="0.5" branches-valid="0" branches-covered="0" branch-rate="0" timestamp="%s" complexity="0" version="0.1">
+    <sources>
+        <source>%s</source>
+    </sources>
+    <packages>
+        <package name="BankAccount" line-rate="1" branch-rate="1" >
+            <classes>
+                <class name="BankAccount" filename="%sBankAccount.php" line-rate="1" branch-rate="1" >
+                    <methods>
+                        <method name="getBalance" hits="2" signature="()V">
```

Comment:
I still need to read up on what that `signature="()V"` stands for exactly, and what it should be set to in terms of PHP coverage reporting.

Reply:
Oh wait the coverage data provides some signature, I presume we can use that here if it is provided.